rpcq/_client.py

```diff
@@ -172,6 +173,9 @@ def call(self, method_name: str, *args, rpc_timeout: float = None, **kwargs):
             else:
                 _log.debug('Discarding reply: %s', reply)
 
+        for warning in reply.warnings:
+            warn("{}: {}".format(warning.kind, warning.body))
```

Comment:
:art: preference for interpolating f-strings

Reply:
This is my preference too, but `.format` really outweighs the `f`-strings in RPCQ. I'll change this one but someone else should change the rest.